docs/intro/install.txt

```diff
@@ -19,13 +19,12 @@ database called SQLite_ so you won't need to set up a database just yet.
 Get the latest version of Python at https://www.python.org/downloads/ or with
 your operating system's package manager.
 
-You can verify that Python is installed by typing ``python`` from your shell;
+You can verify that Python is installed by typing ``python3`` from your shell;
```

Comment:
Hi @pauloxnet. I think this change was rejected before in https://github.com/django/django/pull/16205#pullrequestreview-1157989432. Do you have an idea here that you think might merit reopening the issue?